Index: ChangeLog
===================================================================
RCS file: /home/stb/products/newscache-new/my_cvs/NewsCache/ChangeLog,v
retrieving revision 1.80
retrieving revision 1.81
diff -u -r1.80 -r1.81
--- ChangeLog	28 Sep 2004 19:10:47 -0000	1.80
+++ ChangeLog	8 Nov 2004 18:21:27 -0000	1.81
@@ -1,3 +1,8 @@
+2004-11-03  Herbert Straub <herbert@linuxhacker.at>
+	* configure.in bugfix: configuration failure without configuration
+	option --prefix. Thanks for reporting: Make Harding <mvh@ix.netcom.com>
+	and Tilman Linneweh <arved@freebsd.org>.
+
 2004-09-23  Herbert Straub <herbert@linuxhacker.at>
 	* improved: error messages in class NVcontainer to detect database
 	file corruption.
Index: THANKS
===================================================================
RCS file: /home/stb/products/newscache-new/my_cvs/NewsCache/THANKS,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- THANKS	24 Oct 2003 13:23:34 -0000	1.3
+++ THANKS	8 Nov 2004 18:21:27 -0000	1.4
@@ -4,7 +4,9 @@
 
 * Carles Arjona <nospammer@bigfoot.com> RPM Packager
 
-* Tilman Linneweh <tilman@arved.de> FreeBSD Port
+* Tilman Linneweh <tilman@arved.de> FreeBSD Port, various bug reports
+
+* Mike Harding <mvh@ix.netcom.com> various bug reports
 
 * "miguel a.l. paraz" <map@iphil.net> for the hosts_access patch
 
Index: configure.in
===================================================================
RCS file: /home/stb/products/newscache-new/my_cvs/NewsCache/configure.in,v
retrieving revision 1.44
retrieving revision 1.45
diff -u -r1.44 -r1.45
--- configure.in	28 Sep 2004 19:10:47 -0000	1.44
+++ configure.in	8 Nov 2004 18:21:27 -0000	1.45
@@ -100,7 +100,7 @@
 AC_DEFINE(PACKAGE_UPDATENEWS,"updatenews")
 
 dnl Set SYSCONFDIR in config.h.
-AC_DEFINE_UNQUOTED(SYSCONFDIR,"${sysconfdir}")
+AC_DEFINE_UNQUOTED(SYSCONFDIR,"`eval echo "${sysconfdir}"`")
 
 dnl ****************************************************************
 dnl Checks for programs.
