--- NewsCache-1.2rc6/etc/newscache.auth-dist.orig	2005-06-24 21:38:46.158195544 +0200
+++ NewsCache-1.2rc6/etc/newscache.auth-dist	2005-06-24 21:39:00.464020728 +0200
@@ -1,8 +1,8 @@
 # This are commentar lines
 # You can use plaintext passwords, if the newscache is configured with
 # the option --enable-plainpass
-user1:pass1:at.linux:at.linux:at.linux:post
-user2:pass2:de.linux:de.linux:de.linux:post
-user3:xtKlYxMWGiBYg:de.linux:de.linux:de.linux:post
+# user1:pass1:at.linux:at.linux:at.linux:post
+# user2:pass2:de.linux:de.linux:de.linux:post
+# user3:xtKlYxMWGiBYg:de.linux:de.linux:de.linux:post
 # Encrypted password can be created with the htpasswd command
 # from the apache project or you use a perl script (Google).
--- NewsCache-1.2rc6/etc/newscache.conf-dist.orig	2005-06-24 21:32:22.770479336 +0200
+++ NewsCache-1.2rc6/etc/newscache.conf-dist	2005-06-24 21:38:29.102788360 +0200
@@ -63,11 +63,11 @@
 ################################################################
 ## User and group under which NewsCache should be run
 ################################################################
-Username news
-Groupname news
+Username newscache
+Groupname newscache
 
 # Put in here the local newscache administrator
-Admin tom@infosys.tuwien.ac.at
+Admin news@localhost
 
 ################################################################
 ## Client-access configuration 
@@ -128,46 +128,46 @@
     Authentication unix::::debug
     # all authenticated clients are able to use the xdebug command
   }
-  Client .tuwien.ac.at {
-    allow read post
-    # list all, except for alt.* newsgroups, allow users to 
-    # read at.* and to post to at.*
-    List *,!alt.*
-    Read at.*
-    PostTo at.*
-    # allow users to gain additional privileges by
-    # authenticating based on a password file
-    # the format of the file is username:password:read:postto
-    Authentication file:/usr/local/etc/tuwien.pass
-  }
-  Client 192.168.0.0/255.255.0.0 {
-    allow read post
-    # deny access to all newsgroups
-    List !*
-    Read !*
-    PostTo !*
-    # allow users to gain additional privileges by
-    # authenticating using standard passwd semantics
-    # allow to read * and to post to at.* after successful
-    # authentication
-    Authentication unix:*:at.*
-  }
-  Client 192.169.0.0/255.255.0.0 {
-    allow read
-    # deny access to all newsgroups
-    List at.*
-    Read at.*
-    PostTo !*
-    Authentication unix:*:*:*:debug,post
-    # allow users to gain additional privileges by
-    # authenticating using standard passwd semantics
-    # allow to read * and to post to * and can use the xdebug 
-    # command after successful authentication. 
-    # Unauthenticated clients can only read from at.* groups.
-  }
-  Client .aol.com {
-    allow none
-  }
+#  Client .tuwien.ac.at {
+#    allow read post
+#    # list all, except for alt.* newsgroups, allow users to 
+#    # read at.* and to post to at.*
+#    List *,!alt.*
+#    Read at.*
+#    PostTo at.*
+#    # allow users to gain additional privileges by
+#    # authenticating based on a password file
+#    # the format of the file is username:password:read:postto
+#    Authentication file:/usr/local/etc/tuwien.pass
+#  }
+#  Client 192.168.0.0/255.255.0.0 {
+#    allow read post
+#    # deny access to all newsgroups
+#    List !*
+#    Read !*
+#    PostTo !*
+#    # allow users to gain additional privileges by
+#    # authenticating using standard passwd semantics
+#    # allow to read * and to post to at.* after successful
+#    # authentication
+#    Authentication unix:*:at.*
+#  }
+#  Client 192.169.0.0/255.255.0.0 {
+#    allow read
+#    # deny access to all newsgroups
+#    List at.*
+#    Read at.*
+#    PostTo !*
+#    Authentication unix:*:*:*:debug,post
+#    # allow users to gain additional privileges by
+#    # authenticating using standard passwd semantics
+#    # allow to read * and to post to * and can use the xdebug 
+#    # command after successful authentication. 
+#    # Unauthenticated clients can only read from at.* groups.
+#  }
+#  Client .aol.com {
+#    allow none
+#  }
   Default {
     allow none
   }
@@ -206,29 +206,36 @@
 #   To switch off, prepend the command with not-
 #   Support commands: listgroup, over
 NewsServerList {
-  Server news.tuwien.ac.at nntp {
-    Retries 3
+   Server news.tpi.pl nntp {
+    Retries 5
     Read *
     PostTo *
-    GroupTimeout 150
-    User joe
-    Password secret
-    BindFrom 192.168.100.1
-    Options cached 
-    Commands not-over
-  }
-  Server news.wu-wien.ac.at nntp {
-    Retries 1
-    Read at.wu-wien.*
-    PostTo .none.
-    GroupTimeout 300
-    BindFrom somehost.tuwien.ac.at
-    Options offline not-setpostflag
-  }
-  NoServer {
-    Read au.*,z-netz.*,maus.*,fido*
-    PostTo au.*
-  }
+    GroupTimeout 150 
+    Options semioffline 
+   }
+#  Server news.tuwien.ac.at nntp {
+#    Retries 3
+#    Read *
+#    PostTo *
+#    GroupTimeout 150
+#    User joe
+#    Password secret
+#    BindFrom 192.168.100.1
+#    Options cached 
+#    Commands not-over
+#  }
+#  Server news.wu-wien.ac.at nntp {
+#    Retries 1
+#    Read at.wu-wien.*
+#    PostTo .none.
+#    GroupTimeout 300
+#    BindFrom somehost.tuwien.ac.at
+#    Options offline not-setpostflag
+#  }
+#  NoServer {
+#    Read au.*,z-netz.*,maus.*,fido*
+#    PostTo au.*
+#  }
 }
 
 ################################################################
@@ -265,7 +272,7 @@
 # All the log files go here, if syslogd is not being used. This
 # depends on how NewsCache has been configured. By default,
 # NewsCache uses syslogd.
-LogDirectory /home/services/newscache/log
+# LogDirectory /home/services/newscache/log
 
 # Log Style
 # All combination of these options are valid:
@@ -284,4 +291,4 @@
 # Each newsgroup listed in this file will be prefetched, 
 # whenever one of its articles is read by a user or whenever 
 # updatenews is executed.
-PrefetchFile /home/services/newscache/etc/prefetch
+# PrefetchFile /home/services/newscache/etc/prefetch
